adapters/handlers/grpc/v1/batch_parse_request.go

```diff
@@ -53,8 +53,12 @@ func BatchFromProto(req *pb.BatchObjectsRequest, getClass func(string) *models.C
 				ObjectArrayProperties:  obj.Properties.ObjectArrayProperties,
 				EmptyListProps:         obj.Properties.EmptyListProps,
 			})
+			class, err := getClass(obj.Collection, obj.Tenant)
```

Comment:
**suggestion:** `authorizedGetClass(...)`  to differentiate between normal getters 


side question: why we don't  have authorizer on the grpc service and we can authz directly here `BatchFromProto` instead of authorize from the caller func